test_clickhouse.troy

```diff
@@ -0,0 +1,52 @@
+define flow clickhouse
```

Comment:
this file should probably not be in the PR :)

Reply:
Correct! I removed it in [fb93fb7](https://github.com/tremor-rs/tremor-runtime/pull/1538/commits/fb93fb75452b35cfd008c5586faa7735d1dfdacd).